Data.js

```diff
@@ -1,100 +0,0 @@
-const movies = [
```

Comment:
Mocks are nice thing to have in an app - don't just delete them, consider keeping them under "src/core/mocks"

Reply:
Added a Data file under this